project_milestone_status/models/project_milestone.py

```
@@ -0,0 +1,32 @@
+            executed_tasks = milestone.task_ids.filtered("stage_id.fold")
+
+            total_planned_hours = sum(milestone.task_ids.mapped("planned_hours"))
+            total_excuted_hours = sum(executed_tasks.mapped("planned_hours"))
+
+            if total_excuted_hours and total_planned_hours:
+                milestone.execution = total_excuted_hours * 100 / total_planned_hours
+            else:
+                milestone.execution = 0
+
+    @api.depends("task_ids")
+    def _compute_dedication(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_compute_dedication`

Add return type annotation: `None`

(ANN202)